tutorials/Plugins.md

```diff
@@ -45,11 +45,11 @@ ping = Command "ping" (noArguments $ \m -> do
     return ())
 ```
 
-This can be further simplified through `sendMessageVoid`, which discards the error for us:
+This can be further simplified through `sendMessage`, which discards the error for us:
```

Comment:
This section needs more cut - the code example just above this no longer works.

Reply:
I've updated this. If it's still a problem would you be able to commit your own rewrite?